src/ssr-carousel-dots.vue

```diff
@@ -5,10 +5,13 @@
 .ssr-carousel-dots
 	button.ssr-carousel-dot-button(
 		v-for='i in pages' :key='i'
-		:aria-label='`Page ${i}`'
+		:aria-label='`Go to page ${i}`'
 		:disabled='isDisabled(i)'
```

Comment:
Did the folks running the audit also want us to get rid of `disabled`?  I'd rather not, but if that is important, I'm curious what they'd recommend for hinting that the button is inactive (clicking shouldn't do anything) because it represents the active state.

Reply:
They didn't call out anything regarding the disabled attr. Just that the actual text didn't describe a function, or purpose. Like what will happen _if you click on it_. 

Maybe I can make that label reactive, and only say "Go to... " if it's not disabled, that would be nice I think

Reply:
As with the other request, I'm adding you as a watcher on the task here: https://app.asana.com/0/1202074446600011/1203557689732170/f

Reply:
@weotch they're also calling out this, which is somewhat related (this one for the arrows), and they're asking for it to be more descriptive

![image](https://user-images.githubusercontent.com/435606/215890854-fc48f0f8-299a-40a7-845d-2295303a74a2.png)

I see a couple of approaches here: 
- add the ability to set the label via prop to the slot (like we're doing with the chevron icon)
- just leave it and say we're not fixing. I say this as an option, because in this example they want it to say something like "Previous Image" rather than "Back", but the carousel can hold anything, not just images